src/Containers/Calendar/Components/SwitchMonthButtons.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { View, ViewStyle, StyleSheet } from "react-native";
+import moment from "moment";
+import { TransparentButton } from "src/Components/Buttons";
+import { ChevronLeft, ChevronRight } from "src/Components/Icons";
+import withTheme, { ThemeProps } from "src/Themes/withTheme";
+
+interface DefaultProps {
+  currentDate: Date;
+}
+
+interface Props extends DefaultProps, ThemeProps {
+  onMonthChange(date: Date): void;
+  style?: ViewStyle;
+}
+
+class SwitchMonthButtons extends React.PureComponent<Props> {
+  static defaultProps = {
+    currentDate: new Date(),
+  };
+
+  public render() {
+    return (
+      <View style={[styles.container, this.props.style]}>
+        <TransparentButton
+          style={{ width: undefined }}
+          onPress={this.handlePreviousMonthPress}
+          accessibilityLabel="prevMonth">
+          <ChevronLeft color={this.props.colors.defaultText} />
+        </TransparentButton>
+        <TransparentButton
+          style={{ width: undefined }}
+          onPress={this.handleNextMonthPress}
+          accessibilityLabel="nextMonth">
+          <ChevronRight color={this.props.colors.defaultText} />
+        </TransparentButton>
+      </View>
+    );
+  }
+
+  private handlePreviousMonthPress = () => {
+    const date = new Date(
+      moment(this.props.currentDate)
+        .add(-1, "month")
+        .startOf("month")
+        .toString(),
```

Comment:
If you need to transfer Moment object to Date object, there is a special `.toDate()` method

https://momentjs.com/docs/#/displaying/as-javascript-date/

Reply:
Still exists at `src/Containers/Calendar/Components/MonthCalendar/MonthCalendar.tsx:33`